packages/nextjs-template/components/DendronSEO.tsx

```diff
@@ -47,6 +47,9 @@ export default function DendronSEO() {
   const seoProps = NoteUtils.getSEOProps(noteActive);
   const title = seoProps.title;
   const description = seoProps.excerpt || config.site.description;
+  const images = seoProps.image
+    ? [{ url: seoProps.image, alt: seoProps.imageAlt }]
```

Comment:
will there be a problem if `url` is set and `imageAlt` is undefined?

Reply:
Looks like it does. I followed `next-seo` type declarations and did this. But now i dug into the [OpenGraph Sepc](https://ogp.me/#structured) and the alt is required.

Should we move the image from `frontMatter.custom` to `frontMatter.image?.` and change image from `string` to `{ url: string, alt: string}`?

Reply:
Or using the note title as the alt prop's fallback is easier?

Reply:
Adding this as an actual property to frontmatter sounds like a better design to me. Here's some docs on how to go about adding it: https://wiki.dendron.so/notes/3c80629b-4048-48f0-bdfd-352645bda2ec.html#adding-a-new-frontmatter-property

Reply:
@kevinslin and @SeriousBug please take a look, i've added the `image` as an optional property to the frontmatter.